mars/services/storage/spill.py

```diff
@@ -72,36 +74,91 @@ def unpin_data(self, key):
         if self._pinned_keys[key] <= 0:
             del self._pinned_keys[key]
 
+    @property
+    def spillable_size(self):
```

Comment:
Since this is a non-trivial operation, better to create a counterpart function instead of property.